backend/app/services/git_skill/providers/base.py

```
@@ -0,0 +1,159 @@
+                response = client.get(url, headers=headers if headers else None)
+
+                self._log_api_response(url, response.status_code)
+
+                self._handle_api_errors(response)
+
+                data = response.json()
+                return data.get(
+                    self.get_default_branch_key(), self.get_default_branch_fallback()
+                )
+        except httpx.RequestError as e:
+            logger.error(f"{self.name} API request failed: {str(e)}, url={url}")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)